engine/src/flutter/lib/ui/painting/display_list_deferred_image_gpu_impeller_unittests.cc

```diff
@@ -0,0 +1,103 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/common/graphics/texture.h"  // Include for TextureRegistry
+#include "flutter/display_list/dl_builder.h"
+#include "flutter/fml/synchronization/waitable_event.h"
+#include "flutter/fml/thread.h"
+#include "flutter/lib/ui/painting/display_list_deferred_image_gpu_impeller.h"
+#include "flutter/lib/ui/painting/testing/mocks.h"
+#include "flutter/testing/testing.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "impeller/core/texture_descriptor.h"
+#include "impeller/renderer/testing/mocks.h"
+
+namespace flutter {
+namespace testing {
+namespace {
+
+void PostTaskSync(const fml::RefPtr<fml::TaskRunner>& task_runner,
```

Comment:
Remove the local version of this function and use the one from `#include "flutter/testing/post_task_sync.h"`

Reply:
done